src/main/java/org/webcurator/core/harvester/coordinator/HarvestAgentListener.java

```diff
@@ -33,12 +35,11 @@
     void heartbeat(HarvestAgentStatusDTO aStatus);
 
     /**
-     *
-     * @param haHost
-     * @param haPort
-     * @param haService
+     * Recover connection
```

Comment:
This is a potentially confusing comment: this method recovers crawl jobs

Reply:
Yes. I tried to refine as this:
 * The <code>HarvestAgentListener</code> is a service that 
 * listens for messages from a <code>Crawler</code> and
 * forward the messages to a <code>Store</code> and a
 * <code>Webapp</code>.